contest/templates/contest/contestArchive.html

```diff
@@ -87,35 +103,16 @@
           </a>
 
           {% else %}
```

Comment:
I prefer to comment whole else part(including the else tag) until virtual contest functionality completed.